.travis.yml

```diff
@@ -1,4 +1,21 @@
-language: haskell
+
+env:
+ - GHCVER=7.6.3 CABALVER=1.18
+ - GHCVER=7.8.4 CABALVER=1.18
+ - GHCVER=7.10.1 CABALVER=1.22
+ - GHCVER=head  CABALVER=head
```

Comment:
Wouldn't matter, but is it intentional that there are 2 spaces between `head` and `CABALVER` on this line and on line 11?